jwst/saturation/saturation.py

```diff
@@ -48,22 +49,22 @@ def flag_saturation(input_model, ref_model, n_pix_grow_sat):
         the GROUPDQ array
     """
 
-    data = input_model.data
-
     # Create the output model as a copy of the input
     output_model = input_model.copy()
+    input_model.close()
```

Comment:
Why copy, just to close?  Why not just continue to use `input_model`?

Reply:
thank you for catching this!